docroot/modules/humsci/hs_paragraph_types/modules/hs_row/modules/hs_row_display/hs_row_display.module

```diff
@@ -8,6 +8,20 @@
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
 
+/**
+ * Implements template_preprocess_paragraph()
```

Comment:
I think recall something about comments needing to end with. `.`.  Maybe? 🤔 See lines 26 and 35. 
Maybe it was in a theme settings file that we needed to add periods or there would be an error. I can't remember what throws the error though or why.

Reply:
oh yes. you are right! good catch